test/e2e/e2e.go

```diff
@@ -82,6 +83,18 @@ func autoscalerCM(clients *test.Clients) (*autoscalerconfig.Config, error) {
 	return autoscalerconfig.NewConfigFromMap(autoscalerCM.Data)
 }
 
+// rawCM returns the raw knative config map for the given name
+func rawCM(clients *test.Clients, name string) (*corev1.ConfigMap, error) {
+	return clients.KubeClient.Kube.CoreV1().ConfigMaps(system.Namespace()).Get(
+		name,
+		metav1.GetOptions{})
+}
+
+// patchCM updates the existing config map with the supplied value.
+func patchCM(clients *test.Clients, cm *corev1.ConfigMap) (*corev1.ConfigMap, error) {
+	return clients.KubeClient.Kube.CoreV1().ConfigMaps(system.Namespace()).Update(cm)
+}
```

Comment:
We shouldn't be modifying the System Under Test during e2e test runs.

Reply:
Yes i was not aware of this.
Now modified as per the suggestion Thank you